File: Python/Program/Storage.py
from Component import Component

class Storage(Component):
    def __init__(self, brand="", model="", capacity=0, type_=""):
        super().__init__(brand, model)
        self.capacity = capacity
        self.type = type_

    def set_capacity(self, capacity):
        self.capacity = capacity

    def set_type(self, type_):
        self.type = type_

    def get_capacity(self):
        return self.capacity

    def get_type(self):
        return self.type

File: Java/Program/Storage.java
public class Storage extends Component {
    protected int capacity;
    protected String type;

    public Storage() {}

    public Storage(String brand, String model, int capacity, String type) {
        super(brand, model);
        this.capacity = capacity;
        this.type = type;
    }

    public void setCapacity(int capacity) { this.capacity = capacity; }
    public void setType(String type) { this.type = type; }

    public int getCapacity() { return capacity; }
    public String getType() { return type; }
}
